autogpt/app.py

```diff
@@ -192,6 +193,11 @@ def execute_command(command_name: str, arguments):
             return generate_image(arguments["prompt"])
         elif command_name == "send_tweet":
             return send_tweet(arguments["text"])
+        elif command_name == "send_email":
+            return send_email(
```

Comment:
add  `if use_email:
                return send_email()
          else:
               return("email not set up"
`

Reply:
no this is not necessary and breaks convention on how the other commands work. See also my comment above.